admin/tabs/logging/viewer/LogViewerDisplay.js

```diff
@@ -47,6 +55,17 @@ class LogViewerDisplay extends Component {
         });
     }
 
+    timedRefreshLines = () => {
+        const {model, lastRow} = this,
+            {tabPaneModel, tail} = model,
+            rect = lastRow.value.getBoundingClientRect(),
+            outOfView = rect.bottom > window.innerHeight;
```

Comment:
can't lastRow be null if the file is empty (or just hasn't been rendered).  We should make this null save.